airbyte-cdk/java/airbyte-cdk/typing-deduping/src/main/java/io/airbyte/integrations/base/destination/typing_deduping/FutureUtils.java

```diff
@@ -12,29 +12,40 @@
 
 public class FutureUtils {
 
+  public static final int DEFAULT_TD_THREAD_COUNT = 8;
+
+
   /**
-   * Allow for configuring the number of typing and deduping threads via an enviornment variable in
-   * the destination container.
+   * Allow for configuring the number of typing and deduping threads via an environment variable in the destination container.
    *
    * @return the number of threads to use in the typing and deduping pool
    */
   public static int countOfTypingDedupingThreads(final int defaultThreads) {
     return Optional.ofNullable(System.getenv("TD_THREADS"))
```

Comment:
maybe namespace it for future proofing ? `DESTINATION_TD_THREADS`

Reply:
Type and Dedupe is already kinda namespaced to destiantions, so I think this is ok. Easy to change in the future if needed